src/riak_kv_ttaaefs_manager.erl

```diff
@@ -459,10 +499,11 @@ generate_sendfun(SendClient, NVal) ->
 
 %% @doc
 %% Make a remote client for connecting to the remote cluster
--spec init_client(client_protocol(), client_ip(), client_port())
+-spec init_client(client_protocol(), client_ip(), client_port(),
+                    string()|undefined)
```

Comment:
This seems wrong... wonder why dialyzer does not catch it.
The last argument is used specified to be string() | undefined but below it is used to pass on Credentials, which is a 4-tuple.

Reply:
Yes good catch.  I don't know why dialyzer is missing it either